src/components/Web3Status/index.tsx

```diff
@@ -107,6 +108,22 @@ const Web3StatusConnected = styled(Web3StatusGeneric)<{ pending?: boolean }>`
           pending ? darken(0.1, theme.deprecated_primary1) : darken(0.1, theme.deprecated_bg2)};
     }
   }
+
+  @media only screen and (max-width: ${({ theme }) => `${theme.breakpoint.lg}px`}) {
```

Comment:
what should the width be if its pending? not sure why that's related

Reply:
pending should act as the status bar normally does which is showing the pending transactions and the loading circle.  Otherwise it should just be a circle